assets/training/automl/environments/ai-ml-automl-dnn-text-gpu-ptca/context/Dockerfile

```diff
@@ -1,20 +1,24 @@
-FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py310-torch1131:{{latest-image-tag}}
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu121-py310-torch222:{{latest-image-tag}}
 
 USER root:root
 
 RUN pip install 'azureml-automl-dnn-nlp=={{latest-pypi-version}}'
 RUN pip install 'azureml-defaults=={{latest-pypi-version}}'
 
-RUN pip install torch==1.13.1
+RUN pip uninstall -y onnxruntime
+
+RUN pip install onnx==1.16.2
```

Comment:
If pnnx==1.16.2 is important, we should probably update the pins in  azureml-automl-runtime, azureml-train-automl-runtime and azureml-training-tabular, which currently have 1.16.1

Reply:
The other packages don't need this update. This is specific to onnxruntime-training which is just used in PTCA environment.